docker-compose.deps.yml

```diff
@@ -0,0 +1,9 @@
+services:
```

Comment:
Needs a quick comment at the top to explain what this file is and the fact that it's not used by default.